leasing/importer/area.py

```diff
@@ -466,23 +461,28 @@ def handle_geometry(
     def update_or_create_areas(
         self,
         areas: QuerySet[Area],
-        other_data: OtherData,
+        update_data: UpdateData,
         match_data: MatchData,
         imported_identifiers: List[str],
         error_count: int,
     ) -> Tuple[list, int]:
         try:
-            areas.update_or_create(defaults=dict(other_data), **match_data)
-        except IntegrityError:  # There should only be one object per identifier...
-            ext_id = other_data.pop("external_id", "")
+            areas.update_or_create(defaults=dict(update_data), **match_data)
+        except (
+            MultipleObjectsReturned,
+            IntegrityError,
+        ):  # There should only be one object per identifier...
+            ext_id = (
+                update_data["external_id"] if "external_id" in update_data else None
```

Comment:
Alternative way is to use `get()` with a default value, see: https://docs.python.org/3/library/stdtypes.html#dict.get